windows/d2-installer.nsi

```diff
@@ -355,26 +375,49 @@ Function DetectVSAndSDK
     StrCpy $VCPath $0
     StrCpy $VCVer $1
 
-    ReadRegStr $0 HKLM "Software\Microsoft\Windows Kits\Installed Roots" "KitsRoot81"
+    ; detect ucrt
+    ReadRegStr $0 HKLM "Software\Microsoft\Windows Kits\Installed Roots" "KitsRoot10"
+    IfErrors done done_ucrt
+
+    done_ucrt:
+    StrCpy $UCRTPath $0
+
+      StrCpy $UCRTVersion ""
+      FindFirst $0 $1 $UCRTPath\Lib\*.*
+      loop_ff:
+        StrCmp $1 "" done_ff
+        StrCpy $UCRTVersion $1 ; hoping the directory is retrieved in ascending order (done by NTFS)
+        FindNext $0 $1
+        Goto loop_ff
+      done_ff:
+      FindClose $0
+
+    done:
+
+    !insertmacro _DetectSDK "Windows Kits\Installed Roots" "KitsRoot10" "lib\${UCRTVersion}\um\x64"
```

Comment:
Fixed, though it should not make a big difference as file systems on Windows are not case sensitive.